<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FHIRReader</class>
 <widget class="qMRMLWidget" name="FHIRReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>552</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="connectionCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Connection Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>FHIR Server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="FhirServerLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="DICOMLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DICOMweb Server</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>DICOMweb Status</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="DICOMStatusLabel">
        <property name="text">
         <string>Not Connected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loadPatientsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Connect and Load Patients</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Patient Browser</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="PatientListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Observation Browser</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="ObservationListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>DICOM Browser</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="DICOMTreeWidget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
